corehq/apps/toggle_ui/views.py

```diff
@@ -263,6 +267,16 @@ def _enable_dependencies(request_username, static_toggle, item, namespace, is_en
             _set_toggle(request_username, dependency, item, namespace, is_enabled)
 
 
+def _handle_attendance_tracking_role(domain, is_enabled):
```

Comment:
Considering Attendance Tracking is already conditional in `roles.js`, is this necessary? Other conditional roles don't use this method of enabling and archiving themselves.

Reply:
To close the loop here, other optional features have permissions associated with them, but Attendance Tracking is the only optional feature that has its own role. This function is required to clean up when the toggle is disabled.